ZeeHomeScreen/Classes/Components/SectionComposite/SectionCompositeViewController.swift

```diff
@@ -71,6 +73,32 @@ import Zee5CoreSDK
         }
     }
     
+    @objc open func screenPickerTabSelected(notification: NSNotification) {
+        if let userInfo = notification.userInfo {
+            if let currentCoomponentModel = currentComponentModel, let from: String = userInfo["from"] as? String, from == currentCoomponentModel.title {
+                pauseAllVideoAds()
+            } else if let currentCoomponentModel = currentComponentModel, let to: String = userInfo["to"] as? String, to == currentCoomponentModel.title {
+                resumeAllVideoAds()
+            }
+        }
+    }
+    
+    private func pauseAllVideoAds() {
```

Comment:
Please move this code to the ad view controller. All instances can listen directly to tab change notifications. No need to involve the section composite view controller.